S23/thofstrand/lowerCase.py

```diff
@@ -0,0 +1,76 @@
+import pandas as pd
+
+
+
+def lowerCase(word, lang):
+    word = word.strip()
+    langStart = lang[:2]
+
+    if "en" == langStart:
+        lower_word = word.lower()
+        return lower_word
+    
+    if "tr" == langStart or 'az' == langStart:
+        
+        if "I" in word:
+            word_list = [*word]
+            indices = [i for i, x in enumerate(word_list) if x == "I"]
+            
+            for x in indices:
+                word_list[x] = 'ı'
+
+            lower_word = combineToString(word_list)
+            lower_word = lower_word.lower()
+            return lower_word
+        else:
+            return word
+        
+    if "ga" == langStart:
+        lowCase = word.lower()
+        caps = ['A','E','I','O','U','Á','É','Í','Ó','Ú']
+        if word[0] == 'n' or word[0] == 't':
+            if word[1] in caps and word[2].isalpha(): #must check isalpha as Õ parses as `O`,`~`
```

Comment:
I quiet didn't understand the 3 letter must be a alpha or not. I think just the 1 letter must be a n or a t and the second letter in caps.

Reply:
Essentially one of the chars ( Õ) that was an input splits into two characters (`O`,`~`) when I am separating the string by characters. I used isalpha to check for this and make sure i am not counting that character as one of the listed ones. I do see another issue as I am not checking the length of the word, so this I can fix!